src/core/filesystem.h

```diff
@@ -98,6 +98,17 @@ Status GetDirectoryFiles(
 /// \return Error status
 Status ReadTextFile(const std::string& path, std::string* contents);
 
+/// Create a local copy of the file (if needed).
```

Comment:
Update this comment and one below to talk about folder, not file.

Reply:
In case of graphdef it is a file and not a folder